src/main/java/net/coderbot/iris/pipeline/DeferredWorldRenderingPipeline.java

```diff
@@ -186,7 +184,7 @@ public DeferredWorldRenderingPipeline(ProgramSet programs) {
 
 		GlStateManager._activeTexture(GL20C.GL_TEXTURE0);
 
-		this.flippedBeforeShadow = ImmutableSet.of();
+		this.flippedAfterBegin = ImmutableSet.of();
```

Comment:
This seems to either be wrong or unnecessary given the assignment below.

Reply:
IIRC compiler complains otherwise, I’ll check later.